catroid/res/values/strings.xml

```diff
@@ -583,6 +583,11 @@
     <string name="brick_when">When tapped</string>
     <string name="brick_wait">Wait</string>
     <string name="brick_when_nfc">When NFC</string>
+    <string name="brick_stop_script">Stop script/s</string>
```

Comment:
Maybe use a dynamic string here (one example you can have a look at is in SpriteListFragment ActionModes)